src/test/java/com/hillel/items_exchange/controller/AdvertisementControllerTest.java

```diff
@@ -45,6 +45,16 @@ void findPaginated_shouldBeThrownValidationException() throws Exception {
         assertTrue(mvcResult.getResponse().getContentAsString().contains("must be greater than or equal to 0"));
     }
 
+    @Test
+    void findPaginatedAsThumbnails_shouldBeThrownValidationException() throws Exception {
```

Comment:
We already have negative test for pagination. Would you mind to create positive test instead and validate the fields?

Reply:
Agree. It must be done. I'll do it.